NicolaeChristof/Rise
# Enhancement leafpickup ui animation

This branch adds the UI animations that trigger once you pick up acorns. There should be an animation for the leaves as well as the hearts. These animations should only trigger when you're not full on health and/or branches.

How to Test:
Collect acorns throughout the different seasons. Make sure your health/branches aren't full when doing so. Also experiment with picking up acorns while a UI animation tween is already happening and make sure the leaves/hearts go to the right places.

Known Bugs:
-Collecting the giant acorn will cause spawn one UI leaf to animate (as opposed to the 5 that the acorn grants). Once the animation is done, however, everything should function normally.
-The sprites for the Fall leaves are a bit crooked. I can rotate the Fall leaf in Illustrator (or Photoshop) once I do the leaves falling animation.
